themes/ss4demo/css/demo.css

```diff
@@ -1,3 +1,35 @@
-.right-content {
-    font-size:var(--font-size-base);
+.homepage-cta-right {
+  border: 0.1rem #949494 solid;
+  padding: 3rem;
+  border-radius: 0.4rem;
+  width: 100%;
+
+  @media (min-width: 992px) {
+    width: 30rem;
+  }
+}
+
+.homepage-cta-right__heading {
+  margin-top: 0;
+  font-size: 2rem;
+  font-weight: var(--font-weight-bold);
+}
+
+.homepage-cta-right__content {
+  font-size: var(--font-size-base);
+  font-weight: var(--font-weight-regular);
+}
+
+.homepage-cta-right__link.button {
```

Comment:
Needed both classes to have more specificity than the `.button` styles so I could override them.